tls/t/test_rsa.c

```diff
@@ -83,7 +82,7 @@ rsa_sign(void)
 	TlsRSACtx *rsa;
 	unsigned char hash[PT_LEN], sig[KEY_LEN];
 
-	EXPECT_FALSE(!(mp = ttls_mpi_pool_create(TTLS_MPOOL_ORDER, GFP_KERNEL)));
+	EXPECT_FALSE(!(mp = ttls_mpi_pool_create(0, GFP_KERNEL)));
```

Comment:
We have `EXPECT_OK` and `EXPECT_NOT_NULL/EXPECT_NULL`.

Reply:
Actually, the crypto unit tests don't use Tempesta FW's unit test framework and mock the macros. Anyway, I added the mock for `EXPECT_NOT_NULL`